### PR TITLE
Party invitation

### PR DESCRIPTION
**Pull request by manojpanta**
Implemented Party invitation using twilio txt service, need to add tokens to heroku env before deployment.

### CODE REVIEW

**Review by teresa-m-knowles — APPROVED** [review]
Saw the texts in Manoj's phone! Way to go!

File: app/views/invitation_mailer/invite.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>Hello has invited you to join Brownfield of Dreams.You can create an account with me <%= link_to 'here', 'https://brownfield-of-dreams-1811.herokuapp.com/register'%></p>
```
Comment (Mackenzie-Frey) [inline app/views/invitation_mailer/invite.html.erb]: I'm confused about this invite_mailer - Brownfield stuff.

File: app/views/invitation_mailer/invite.text.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>Hello has invited you to join Brownfield of Dreams.You can create an account with me <%= link_to 'here', 'https://brownfield-of-dreams-1811.herokuapp.com/register'%></p>
```
Comment (Mackenzie-Frey) [inline app/views/invitation_mailer/invite.text.erb]: This one too.

**Review by Mackenzie-Frey — CHANGES_REQUESTED** [review]
I'd love to see it in action later. As long as the Brownfield of Dreams things are removed, merging this works for me.

File: app/views/invitation_mailer/invite.html.erb (hunk 1 above)
Comment (manojpanta) [reply app/views/invitation_mailer/invite.html.erb]: Just removed those lines

**Comment by manojpanta** [comment]
Merging this to test in heroku

**Pull request merged**